public/app/plugins/panel/heatmap/module.tsx

```diff
@@ -413,5 +414,13 @@ export const plugin = new PanelPlugin<Options, GraphFieldConfig>(HeatmapPanel)
       defaultValue: defaultOptions.exemplars.color,
       category,
     });
+
+    builder.addCustomEditor({
+      id: 'exemplar labels',
+      path: 'exemplars.labels',
+      name: 'Labels',
+      editor: ExemplarLabelEditor,
+      category,
+    });
```

Comment:
Given the exemplar label editor pertains purely to the exemplar tooltip, it might make more sense to display these options underneath the Tooltip editor (potentially on a conditional basis if we detect that exemplars exist? Currently it is not clear that these settings pertain to just the tooltip view

![this mode has to just do with tooltip - why not embed it there for time series : heatmap panels?](https://github.com/grafana/grafana/assets/22381771/79cbd00f-4074-4b6b-ad24-87544ba87fb3)